server/service/src/login.rs

```diff
@@ -379,6 +379,9 @@ fn permissions_to_domain(permissions: Vec<Permissions>) -> HashSet<Permission> {
                 output.insert(Permission::LogQuery);
             }
             // patient
+            Permissions::AddPatients => {
+                output.insert(Permission::PatientMutate);
```

Comment:
What about `Permission::PatientQuery`? there is no matching permission for it, is there? In this case I think we should add it as well as has been done for EditPatientDetails.

Reply:
Have added [6a846b6](https://github.com/openmsupply/open-msupply/pull/2174/commits/6a846b67168acba8c0af47d34f33fdf666c5924e)...